rules/alerting.go

```diff
@@ -109,6 +109,12 @@ type AlertingRule struct {
 	// only after the restoration.
 	restored bool
 
+	// the health of the alerting rule
```

Comment:
Style nit: comments should start upper-cased and end with a period (sentence-style), here and everywhere else.